charts/echo-server/templates/deployment.yaml

```diff
@@ -40,6 +42,12 @@ spec:
             httpGet:
               path: /ping
               port: http
+              httpHeaders: {{ .Values.image.httpHeaders }}
```

Comment:
does this exists?

Reply:
Well spotted! Thank :) Resolve with #27 #28 I have also improve my workflow ;)